common/src/main/java/org/ergoplatform/appkit/JavaHelpers.scala

```diff
@@ -524,6 +535,40 @@ object JavaHelpers {
     val firstPath = org.ergoplatform.wallet.Constants.eip3DerivationPath
     DerivationPath(firstPath.decodedPath.dropRight(1), firstPath.publicBranch)
   }
+
+  type TokenColl = Coll[(Coll[Byte], Long)]
+
+  def checkAllTokensPositive(tokens: TokenColl): TokenColl = {
+    val invalidTokens = tokens.filter(_._2 <= 0)
+    require(invalidTokens.isEmpty, s"All token values should be > 0: ")
+    tokens
+  }
+
+  def subtractTokens(
```

Comment:
Ah comment would help here, especially that this method can take multiple list elements of the same token and combines them by token id would be helpful here.

Reply:
done